geomesa-kafka/geomesa-kafka-09-datastore/src/main/scala/org/locationtech/geomesa/kafka09/GeoMessage.scala

```diff
@@ -0,0 +1,189 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.kafka09
+
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets
+
+import com.typesafe.scalalogging.LazyLogging
+import org.joda.time.Instant
+import org.locationtech.geomesa.features.SerializationOption.SerializationOptions
+import org.locationtech.geomesa.features.kryo.KryoFeatureSerializer
+import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
+
+sealed trait GeoMessage {
```

Comment:
There were alot of places where zkUtils gets used instead of zkClient.  A few other method signature changes as well.  Most (but not all) of the changes can be seen here:

https://github.com/locationtech/geomesa/compare/master...ddseapy:kafka09